client/src/components/CampaignCard/CampaignCard.jsx

```diff
@@ -11,7 +11,9 @@ import ProgressBar from "../ProgressBar/ProgressBar";
 const CapmpaignCard = ({
   status,
   title,
-  user,
+  owner,
```

Comment:
What are you using this data for? I only see changes in the props but not implementing them